### ISSUE TITLE
dlg in attack module: attack fedavg, the loss is still high after searching the dummy data

### ISSUE THREAD

**Issue by Osier-Yi** [alibaba/FederatedScope#339]
**Describe the bug**
When running the provided dlg attack script, the loss of dlg after the max iteration rounds is still very high, and cannot recover the original training data.

**To Reproduce**
Run the provided dlg attack script.

**Expected behavior**
The loss should be close to 0 after several iterations and the original training data can be recovered.
